smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/commit/KafkaThrottledLatestProcessedCommit.java

```diff
@@ -224,6 +224,8 @@ private void startFlushAndCheckHealthTimer() {
      * @return the map of partition -> offset that can be committed.
      */
     private Map<TopicPartition, Long> clearLesserSequentiallyProcessedOffsetsAndReturnLargestOffsetMapping() {
+        cleanupPartitionOffsetStore();
```

Comment:
Move this here since it was being called multiple times unecessarily from within the for loop on like 230